tests/parser/parser.cpp

```diff
@@ -1858,6 +1858,110 @@ TEST(Parser, xmlComments)
     EXPECT_EQ(size_t(0), parser.errorCount());
 }
 
+TEST(Parser, mathWithNamespacesDefinedOnTheMathNode)
+{
+    const std::string input =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/2.0#\" name=\"model_name\">\n"
+        "  <component>\n"
+        "  <math xmlns=\"http://www.w3.org/1998/Math/MathML\" xmlns:cellml=\"http://www.cellml.org/cellml/2.0#\">\n"
+        "    <apply>\n"
+        "      <eq/>\n"
+        "      <ci>b2</ci>\n"
+        "      <apply>\n"
+        "        <times/>\n"
+        "        <cn cellml:units=\"dimensionless\">2.0</cn>\n"
+        "        <ci>d</ci>\n"
+        "      </apply>\n"
+        "    </apply>\n"
+        "  </math>\n"
+        "  </component>\n"
+        "</model>\n";
+
+    libcellml::Parser parser;
+    parser.parseModel(input);
+
+    EXPECT_EQ(size_t(0), parser.errorCount());
+}
+
+TEST(Parser, mathWithNamespacesDefinedOnTheNodeThatUsesNamespace)
+{
+    const std::string input =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/2.0#\" name=\"model_name\">\n"
+        "  <component>\n"
+        "  <math xmlns=\"http://www.w3.org/1998/Math/MathML\">\n"
+        "    <apply>\n"
+        "      <eq/>\n"
+        "      <ci>b2</ci>\n"
+        "      <apply>\n"
+        "        <times/>\n"
+        "        <cn xmlns:cellml=\"http://www.cellml.org/cellml/2.0#\" cellml:units=\"dimensionless\">2.0</cn>\n"
+        "        <ci>d</ci>\n"
+        "      </apply>\n"
+        "    </apply>\n"
+        "  </math>\n"
+        "  </component>\n"
+        "</model>\n";
+
+    libcellml::Parser parser;
+    parser.parseModel(input);
+
+    EXPECT_EQ(size_t(0), parser.errorCount());
+}
+
+TEST(Parser, mathWithNamespacesDefinedOnTheMathNodeNonStandardPrefix)
```

Comment:
What was the intention here? The test name gives the impression that we wanted to test a non-standard prefix for the CellML namespace, which is the case here, but it's not definied on the math node (as claimed in the name of the test), but on the model (as can be seen in the code). So, which one should it be? (I don't think it matters since the most important, here, is probably to test the non-standard prefix?)

Reply:
I think the test should be renamed to `mathWithNonStandardCellMLPrefix`.

Reply:
Sounds good to me. Done in commit d4327175.